src/components/ellipsis-text/styles.ts

```diff
@@ -12,15 +12,11 @@ export const ReactTooltip = styled(Tooltip).attrs({
     border-radius: 8px;
     cursor: context-menu;
 
-    ${({ theme }) => css`
-      background-color: ${theme.colors.white};
-      color: ${theme.colors.blackColorText};
-      box-shadow: ${theme.colors.ellipsisShadow};
-    `}
-  }
-
-  &.custom-arrow {
-    background: transparent;
+      ${({ theme }) => css`
+        background-color: ${theme.colors.white};
+        color: ${theme.colors.blackColorText};
+        box-shadow: ${theme.colors.ellipsisShadow};
```

Comment:
here you can also sort by length for better reading

Reply:
done